cms/envs/test.py

```diff
@@ -264,3 +264,7 @@
 
 # Dummy secret key for dev/test
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+# This is practically unused but needed by the oauth2_provider package, which
+# some tests in common/ rely on.
+OAUTH_OIDC_ISSUER = 'https://example.com/oauth2'
```

Comment:
Even though the tests are skipped, the test module is still loaded, and one of the modules imported by the test module refers to this setting (without using getattr).